evap/staff/importers/base.py

```diff
@@ -216,10 +216,9 @@ def map(self, file_content: bytes):
                     )
                 )
 
-            for row_index in range(self.skip_first_n_rows, sheet.max_row):
-                # We use 0-based indexing, openpyxl uses 1-based indexing.
-                row = [cell.value for cell in sheet[row_index + 1]]
-                location = ExcelFileLocation(sheet.title, row_index)
+            # We use 0-based indexing, openpyxl uses 1-based indexing.
+            for row_index, row in enumerate(sheet.iter_rows(min_row=self.skip_first_n_rows + 1, values_only=True)):
+                location = ExcelFileLocation(sheet.title, row_index + 1)
```

Comment:
If `skip_first_n_rows` is non-zero, `row_index` will be incorrect. To get rid of the `+1` in the second line, I ~~wanted to~~ suggest using `enumerate(..., start=...)`~~, but even better: Let's just not use `values_only` and ask each cell for its `row`.~~ (edit: no, not better, maybe there are no cells in a row)

Should we add a test?

Reply:
Added a test in https://github.com/e-valuation/EvaP/pull/1841/commits/4a6a44658d1e09a45899c0407ddc57d8a8e1b924, maybe check if you think it's worth it.

We never use the mapper with another `skip_first_n_rows` than one. We could also remove that feature, and have 1 hardcoded until we need something different (which likely isn't going to happen -- you could just remove the excess lines using spreadsheet software before uploading for import). I think I'd like that a bit more.